radar/2024-12-01/axiom.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Axiom"
+ring: hold
+quadrant: tools
+featured: true
+---
+
+Axiom is a centralized logging and monitoring platform designed to simplify application and system-level observability. It was trialed as a solution for all-purpose logging and monitoring, complementing other tools like Sentry, which focuses on frontend error logging.
+
+### Previous Benefits of Axiom
+
+- **All-Purpose Logging:** Provided robust capabilities for both application and system-level logging in a centralized platform.
+- **Ease of Use:** Offered a user-friendly interface and flexible querying capabilities, making it accessible for developers and operations teams.
+- **Comprehensive Monitoring:** Addressed a wide range of use cases, including real-time monitoring, long-term log retention, and advanced analytics.
+
+### Reasons for Holding
+
+- **Established Alternatives:** Tools like Sentry, Datadog, and Grafana provide more mature and widely adopted solutions for logging and monitoring.
+- **Standardization Goals:** We are focusing on standardizing observability tools with providers that offer comprehensive ecosystems and better integration capabilities.
+- **Feature Overlap:** While Axiom held promise, its feature set overlaps with more established tools, which already fulfill our monitoring and logging requirements.
+
+### Current Stance at INFO
+
+Axiom has been placed on hold as we prioritize Sentry for frontend error logging and Datadog or Grafana for broader monitoring and observability needs. These tools align better with our long-term goals, offer extensive community support, and provide a more unified approach to monitoring and logging across our systems. Existing setups with Axiom will be maintained, but future observability projects will utilize these established alternatives.
```

Comment:
In practise though we often use Graylog for centralized (backend) logging. E.g. for Dimpact we choose again for Graylog (for our own centralised logging at least). If we really favour Datadog over Graylog we need to put some action to this effect.

Reply:
I think this is something to discuss; they are very different solutions also saas over self deploy...